pkg/configs/db/postgres/postgres.go

```diff
@@ -42,8 +46,32 @@ type dbProxy interface {
 	Prepare(query string) (*sql.Stmt, error)
 }
 
+// dbWait waits for database connection to be established
+func dbWait(db *sql.DB) error {
+	deadline := time.Now().Add(dbTimeout)
+	var err error
+	for tries := 0; time.Now().Before(deadline); tries++ {
+		err = db.Ping()
+		if err == nil {
+			return nil
+		}
+		log.Debugf("db connection not established, error: %s; retrying...", err)
```

Comment:
What about using `WARN` here? Unless `DEBUG` is the default level, it doesn't say why it's stalling